test/sass/functions/sizes.spec.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../node_modules/sass-true/sass/true";
+@import "../../../source/css/sass/functions/sizes";
+@import "../variables_test";
+
+@include test-module("the function get-rem-from-px [Function]") {
```

Comment:
`"get-rem-from-px [Function]"` for consistency.

Reply:
Wonder about `@function get-rem-from-px`?

Reply:
It would mean changing the ones currently describing mixins to `@mixin some-name`. I don't feel strongly about it either way, as long as their descriptions in `test-module`s are in a consistent format.